epub33/common/js/biblio.js

```diff
@@ -33,8 +33,12 @@ var biblio = {
 		"href": "http://www.daisy.org/guidelines/epub/navigable-audio-only-epub3-guidelines"
 	},
 	"EPUB-3": {
-		"title": "EPUB 3",
-		"href": "https://www.w3.org/publishing/epub3/",
+		"title": "EPUB 3 Overview",
```

Comment:
This should be a reference to the latest version of the EPUB core spec, not the overview (/TR/epub). It's used, for example, so the accessibility specification can always refer to the latest release if they're not always updated in sync. Any links the ref is associated with are also supposed to be version independent (but need to go back and check those).

I don't think any specification except the core has ever referenced the overview, so we don't need a "latest" link for it.

Reply:
I changed it. I also removed the list of editors (for now) if we refer to EPUB 3 in general and to its 3.3 incarnation... Maybe we can change that later.